package.json

```diff
@@ -20,10 +20,11 @@
         }
     ],
     "scripts": {
+        "version": "npm run build-all",
```

Comment:
Shouldn't those modified or additional commands not be a separate PR to be aligned with the contribution guidelines from @cgewecke?

Reply:
***NOT*** a good idea to have a script named the same as a discrete step in the NPM release-lifecycle unless you fully intend to follow the documented behavior:
> Run AFTER bumping the package version, but BEFORE commit.
See: https://docs.npmjs.com/misc/scripts

Maybe that's the case, but noting out of caution.

Reply:
Yes thanks that was exactly our intention, it is to build the minified file after bumping package versions but before committing the working state.

Reply:
Hmmm... not entirely sure, but sounds like maybe `"files"` in `package.json` (root or children) could be added / edited; and that `.gitignore` (root or children) could be adjusted to ignore `*.min.js`).

It's been awhile since I looked closely, but if you have `.npmignore` in root or child packages, nuke with prejudice and prefer `"files"` in all child packages' `package.json` — it will save you a world of trouble.

Reply:
Also relate, consider `"prepublishOnly'` in `package.json` scripts.

Reply:
> Hmmm... not entirely sure, but sounds like maybe "files" in package.json (in child packages) could be added / edited; and that .gitignore (root or children) could be adjusted to ignore *.min.js. Main point: if you're specifying a "version" script, something's probably not right.

I agree.. I've added the ``files``property to the ``package.json`` files in ``2.x``... 

> and that .gitignore (root or children) could be adjusted to ignore *.min.js. 

Sadly no.. I've noticed the min file has to be on GitHub because many do use it for their tutorials and courses or of whatever reason :-)

Reply:
@ryanio 
> NOT a good idea to have a script named the same as a discrete step

I would add this command/alias to the required places without defining a ``version`` command because this is for some/many devs a "hidden logic" :D

Reply:
There's more about this in the lerna documentation for [version command lifecycle scripts][1].